enterprise/server/configure.md

```diff
@@ -216,6 +213,68 @@ ephox {
 }
 ````
 
+### `link-checking` (optional)
+
+The Link checker has three configurable settings:
+
+- `enabled`
+- `fallback-to-get`
+- `link-checking.cache`
+
+#### `enabled` (optional)
+
+Used to enable (`true`) or disable (`false`) the Link-checking service. This setting is `true` by default.
+
+For example:
+
+```
+ephox {
+  link-checking {
+    enabled = true
+  }
+}
+```
+
+#### `fallback-to-get` (optional)
+
+{{site.requires_jsscwar_230v}}
+
+The Link-checker normally relies on the `HEAD` response. If this setting is `true`, the link-checker will fall-back to `GET` responses. When `true`, the Link checker can correctly identify working URLs that return non-standard `HEAD` replies. This setting can lead to performance issues when enabled (`true`) and is set to `false` by default.
```

Comment:
I think `this setting` loses the context of "this" too much. Probably restate the setting: 

> if `fallback-to-get` is true ....

Reply:
>  This setting can lead to performance issues when enabled (`true`) and is set to `false` by default.

Would you normally give more justification than this? Or is this sufficient? Something like, 

Also, be careful about saying "performance issues" without specifying the client or the server. Performance issues on the server for something like link-checking will be noticeable to the integrator, performance issues on the client to the user. It won't affect the performance of the client (aside from how quickly they see underlines turn up)

Reply:
> fallback (or fall-back, whatever we use consistently) to `GET` is also a bit misleading. It will do the `HEAD`, then when that fails for certain reasons, will do a `GET` as well to double-check. `fall-back` in the context you have it makes it sound a bit more like it just always uses GET from the beginning. Probably a problem with the name of the parameter :)

Reply:
I've made a few changes to address these issues. 
Regarding:
> Would you normally give more justification than this? Or is this sufficient?

Sometimes. Bit of a balancing act between what users need to know and TMI. Keeping info minimal is important to save users time, and as the description grows, the likelihood of people reading it decreases.

Reply:
That makes sense.